Quadriga/src/main/java/edu/asu/spring/quadriga/rest/open/NodeInfoController.java

```diff
@@ -111,4 +114,11 @@ public int compare(TextPhrase o1, TextPhrase o2) {
         return new ResponseEntity<String>(result.toString(), HttpStatus.OK);
 
     }
+
+    public void appendCall(JSONObject occurance, TextOccurance occur) throws JSONException {
```

Comment:
method names should say what they actually do (on a semantic level, not on a technical level), otherwise they are difficult to reuse if nobody knows what they are doing by looking at their name.